openjdk-testing/src/main/java/org/conscrypt/testing/TestClient.java

```diff
@@ -39,8 +37,8 @@ public TestClient(SSLSocket socket) {
     public void start() {
         try {
             socket.startHandshake();
-            input = new BufferedInputStream(socket.getInputStream());
-            output = new BufferedOutputStream(socket.getOutputStream());
+            input = socket.getInputStream();
```

Comment:
In my own testing, buffer size has had an impact on performance due to the JNI layer. (This is in fact what I was testing.)

By wrapping the streams this way, you're effectively fixing the buffer size to the implementation default (likely 8192 in most cases).